spec.html

```diff
@@ -15479,8 +15479,8 @@
           <emu-see-also-para op="ContainsDuplicateLabels"></emu-see-also-para>
           <emu-grammar>IfStatement : `if` `(` Expression `)` Statement `else` Statement</emu-grammar>
           <emu-alg>
-            1. Let _hasUndefinedLabels_ be ContainsDuplicateLabels of the first |Statement| with argument _labelSet_.
-            1. If _hasUndefinedLabels_ is *true*, return *true*.
+            1. Let _hasDuplicateLabels_ be ContainsDuplicateLabels of the first |Statement| with argument _labelSet_.
+            1. If _hasDuplicateLabels_ is *true*, return *true*.
```

Comment:
I'd prefer to use the variable name `hasDuplicates` instead of `hasDuplicateLabels`, because that way the naming matches 13.2.2 Static Semantics: ContainsDuplicateLabels, 13.12.2 Static Semantics: ContainsDuplicateLabels, 13.15.2 Static Semantics: ContainsDuplicateLabels, etc.